src/tests/unittests/asn1.c

```diff
@@ -47,6 +47,61 @@ static void torture_large_oid(void **state)
 	assert_int_equal(rv, SC_ERROR_NOT_SUPPORTED);
 }
 
+static void torture_oid(void **state)
+{
+	/* (without the tag and length {0x06, 0x06}) */
+	/* Small OIDs */
+	u8 small[] = {0x01, 0x02, 0x03, 0x04, 0x05, 0x06};
+	/* Limit what we can fit into the first byte */
+	u8 limit[] = {0x7F};
+	/* The second octet already oveflows to the second byte */
+	u8 two_byte[] = {0x81, 0x00};
+	/* Missing second byte, even though indicated with the first bit */
+	u8 missing[] = {0x81};
```

Comment:
Something like this, but for the 3rd subidentifier may be also interesting

Reply:
*bug